src/test/kotlin/com/atkinsondev/opentelemetry/build/OpenTelemetryBuildPluginSpansIntegrationTest.kt

```diff
@@ -0,0 +1,188 @@
+package com.atkinsondev.opentelemetry.build
+
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.json.Json
+import okhttp3.OkHttpClient
+import okhttp3.Request
+import org.gradle.testkit.runner.GradleRunner
+import org.gradle.testkit.runner.TaskOutcome
+import org.junit.jupiter.api.Assertions.assertLinesMatch
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.io.TempDir
+import org.testcontainers.containers.GenericContainer
+import org.testcontainers.containers.wait.strategy.Wait
+import strikt.api.expectThat
+import strikt.assertions.*
+import java.io.File
+import java.nio.file.Path
+
+class OpenTelemetryBuildPluginSpansIntegrationTest {
+    private val json =
+        Json {
+            ignoreUnknownKeys = true
+        }
+
+    @Serializable
```

Comment:
Used `kotlinx-serialization` is it is very easy to use in my opinion.

It's the first tool that I try when having to work with JSON.

Reply:
Using Kotlin's serialization works for me 👍